deployment/src/main/java/io/quarkiverse/loggingui/quarkus/logging/ui/deployment/LoggingUiProcessor.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkiverse.loggingui.quarkus.logging.ui.deployment;
+
+import java.util.function.BooleanSupplier;
+
+import io.quarkiverse.loggingui.quarkus.logging.ui.LoggerUiRecorder;
+import io.quarkus.deployment.Capabilities;
+import io.quarkus.deployment.Capability;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.smallrye.openapi.deployment.spi.AddToOpenAPIDefinitionBuildItem;
+import io.quarkus.vertx.http.deployment.RouteBuildItem;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+class LoggingUiProcessor {
+    private static final String FEATURE = "quarkus-logging-ui";
+
+    static class OpenAPIIncluded implements BooleanSupplier {
+        LoggingUiConfig config;
+
+        public boolean getAsBoolean() {
+            return config.openapiIncluded;
+        }
+    }
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FEATURE);
+    }
+
+    @Record(ExecutionTime.STATIC_INIT)
+    @BuildStep
+    void buildExecutionService(BuildProducer<RouteBuildItem> routeProducer,
+            LoggingUiConfig loggingUiConfig,
+            LoggerUiRecorder recorder) {
+
+        Handler<RoutingContext> loggerHandler = recorder.loggerHandler();
```

Comment:
Can you please explain the difference between using the recorder and doing a plain object instantiation?

Reply:
I'll look into this. All other extensions that I know where you can set the path, it's on build time only. (JAX-RS, GraphQL, Health etc) so maybe we do not need it. Users can still have different paths for different environment (using config %dev%. etc).

Reply:
Let's concentrate the discussion on the other thread.